slash_commands/nft.js

```diff
@@ -0,0 +1,75 @@
+const { SlashCommandBuilder } = require('discord.js');
+const { red, cyber } = require('../jsons/colors.json');
+const bHelper = require("../helpers/bloons-general.js");
+
+const builder = new SlashCommandBuilder()
+.setName('nft')
+.setDescription('Calculate Quincy Action Figure buy/sell price in Shol Geraldo\'s shop')
+.addIntegerOption(option => option.setName('unlock_round').setDescription("Start of round when the action figure is unlocked").setRequired(true).setMinValue(1))
+.addIntegerOption(option => option.setName('start_round').setDescription("Round when the action figure is bought").setRequired(true).setMinValue(1))
+.addIntegerOption(option => option.setName('end_round').setDescription("Round when the action figure is sold").setRequired(true).setMinValue(1))
+.addStringOption(option => option
+    .setName('difficulty')
+    .setDescription("Gamemode difficulty")
+    .setRequired(true)
+    .addChoices(
+        { name: 'Easy (Primary only, Deflation)', value: 'easy' },
+        { name: 'Medium (Military only, Reverse, Apopalypse)', value: 'medium' },
+        { name: 'Hard (Magic only, Double HP)', value: 'hard' },
+        { name: 'Impoppable', value: 'impoppable' }
+    )
+)
+.addBooleanOption(option => option
+    .setName('better_sell_deals')
+    .setDescription("Whether the Better Sell Deals MK is enabled")
+    .setRequired(false)
+);
```

Comment:
I wonder if this should be default on? Also I think having an optional boolean option is kinda redundant because one of the values is going to be the same as if you skipped the argument (`better_sell_deals: false` in this case)

It's up to you but maybe you could call this "turn_off_mk" with one string option of "yes". The description can be "Whether to disable the Better Sell Deals MK".